rasa/core/test.py

```diff
@@ -791,23 +789,6 @@ async def _predict_tracker_actions(
                         "confidence": prediction.max_confidence,
                     }
                 )
-
-        elif use_e2e and isinstance(event, UserUttered):
```

Comment:
wait - this is actually a crucial piece as this handles the end-to-end conversation testing  (which does Core + NLU predictions)